### PR TITLE
Make User Registration 'remove' button Red

### PR DESCRIPTION
**Pull request by thesuss**
PT Story: https://www.pivotaltracker.com/story/show/151225930

**_I dont know what the db/structure.sql is?_** 

Changes proposed in this pull request:
1. Adds btn-danger class to button in form
2. Enforces css of that on the button

Screenshots (Optional):
skarmavbild 2017-09-29 kl 17 44 51

Ready for review:
@weedySeaDragon @RobertCram @patmbolger

### CODE REVIEW

**Comment by weedySeaDragon** [comment]
You missed my question to Robert about the `db/structure.sql` file in the first part of the meeting on Thurs. :-)  
This is a file created by Rails for the database since we are now using a Postgres-specific thing with our database. (the member number sequence that Robert did.)  If you are using some database feature that is vendor specific (as we are now), Rails will also create this file, in addition to schema.db.  So now we need to have both files (schema & structures.sql) checked in to git.

**Review by weedySeaDragon — CHANGES_REQUESTED** [review]
I'd like to see the title of this PR be more informative and specific. Something like "Make User Registration 'remove' button Red".

**Comment by patmbolger** [comment]
The color styling looks good.  In addition, I think we should improve the alert message, to say something like "Are you sure your want to DELETE your account?".

screen shot 2017-09-30 at 5 14 44 pm

File: app/assets/stylesheets/style.css [hunk 1]
```diff
@@ -527,7 +527,14 @@ textarea {
     margin-top:45px !important;
     text-align:center;
 }
-
+.btn-danger {
```
Comment (weedySeaDragon) [inline app/assets/stylesheets/style.css:530]: This is already defined in bootstrap-sass in the `/assets/stylesheets/bootstrap/_buttons.scss` file that defines the style:
```
// Danger and error appear as red
.btn-danger {
  @include button-variant($btn-danger-color, $btn-danger-bg, $btn-danger-border);
}
```

and the `assets/stylesheets/bootstrap/_variables.scss` file that defines the actual colors:
```
$brand-danger:          #d9534f !default;

...

$btn-danger-color:               #fff !default;
$btn-danger-bg:                  $brand-danger !default;
$btn-danger-border:              darken($btn-danger-bg, 5%) !default;
```

We should just use those styles.

**Review by weedySeaDragon — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/style.css (hunk 1 above)
Comment (thesuss) [reply app/assets/stylesheets/style.css:530]: I know this, but this is overwritten by wpcf7 for this particulat button, thats why I add it again with the !important addition.

Comment (weedySeaDragon) [reply app/assets/stylesheets/style.css:530]: aha.  (I figured that you must know that, but perhaps had just forgotten momentarily or something.)

**Review by weedySeaDragon — APPROVED** [review]

**Comment by weedySeaDragon** [comment]
update and squash and merge away, @thesuss ! :-)

**Pull request merged**